conda/recipes/versions.yaml

```
@@ -11,7 +11,7 @@ nccl_version:
 numba_version:
   - '>=0.57'
 numpy_version:
-  - '>=1.23,<2.0a0'
+  - '>=2,<3.0a0'
```

```suggestion
  - '>=1.23,<3.0a0'
```

oops, forgot to set the floor back down to `>=1.23`.

But good to see everything built successfully when it was forced to `numpy>=2`!

https://github.com/rapidsai/integration/actions/runs/11109542778/job/30865151500?pr=718

---

Was about to ask. Assumed this was for testing purposes